### PR TITLE
Save acsp data to Mongo

### PR DESCRIPTION
**Pull request by sardhani-ch**
__Short description outlining key changes/additions__

__JIRA Ticket Number__

### Type of change

* [ ] Bug fix
* [ ] New feature
* [ ] Breaking change
* [ ] Infrastructure change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/acsp/model/AMLSupervisoryBodies.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package uk.gov.companieshouse.acsp.model;
+
+public enum AMLSupervisoryBodies {
+    SOLE_TRADER("Sole Trader"),
```
Comment (sampankumar) [inline src/main/java/uk/gov/companieshouse/acsp/model/AMLSupervisoryBodies.java:4]: AMLSupervisory bodies would be something like HMRC, FDA etc. check with Benedict to get the list.

File: src/main/java/uk/gov/companieshouse/acsp/model/AcspData.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
+package uk.gov.companieshouse.acsp.model;
```
Comment (sampankumar) [inline src/main/java/uk/gov/companieshouse/acsp/model/AcspData.java:1]: use lombok if possible

File: src/main/java/uk/gov/companieshouse/acsp/model/Address.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
+package uk.gov.companieshouse.acsp.model;
+
```
Comment (sampankumar) [inline src/main/java/uk/gov/companieshouse/acsp/model/Address.java:2]: Same as above lets try to use lombok

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Pull request merged**